newIDE/app/src/UI/Dialog/index.js

```diff
@@ -106,29 +122,30 @@ export const DialogPrimaryButton = RaisedButton;
  * A enhanced material-ui Dialog that can have optional secondary actions
  * and no margins if required.
  */
-const Dialog = (props: Props) => {
-  const {
-    onApply,
-    secondaryActions,
-    actions,
-    open,
-    onRequestClose,
-    maxWidth,
-    noMargin,
-    title,
-    children,
-    flexColumnBody,
-    flexBody,
-    fullHeight,
-    noTitleMargin,
-    id,
-    cannotBeDismissed,
-  } = props;
-
+const Dialog = ({
+  onApply,
+  secondaryActions,
+  isDangerous,
```

Comment:
You use this prop only for the DeleteDialog if i'm not mistaken?
the naming is a bit confusing, are we planning to use it for further dialogs?
If so, should it be something like "isNewDialogDesign"?

Reply:
We could use it for other dialogs yes.
There is already a notion of danger in Material ui to set the usual red color so the naming seemed ok.

Reply:
alright 👍